maintenance/passport-azure-ad/lib/jsonWebToken.js

```diff
@@ -53,176 +36,200 @@ var hasCommonElem = (array1, array2) => {
-    throw new Error('callback must be provided in jsonWebToken.verify');
-  if (typeof callback !== 'function')
-    throw new Error('callback in jsonWebToken.verify must be a function');
+    throw new Error("callback must be provided in jsonWebToken.verify");
+  if (typeof callback !== "function")
+    throw new Error("callback in jsonWebToken.verify must be a function");
   if (!options)
-    throw new Error('options must be provided in jsonWebToken.verify');
+    throw new Error("options must be provided in jsonWebToken.verify");
 
-  // check jwtString and PEMKey are provided. Since jwtString and PEMKey are generated, this is
-  // a non-server error, we shouldn't throw, we just give the error back and let authentication fail.
-  if (!jwtString || jwtString === '')
-    return done(new Error('jwtString must be provided in jsonWebToken.verify'));
-  if (!PEMKey || PEMKey === '')
-    return done(new Error('PEMKey must be provided in jsonWebToken.verify'));
+  /*
+   * check jwtString and PEMKey are provided. Since jwtString and PEMKey are generated, this is
+   * a non-server error, we shouldn't throw, we just give the error back and let authentication fail.
+   */
+  if (!jwtString || jwtString === "")
+    return done(new Error("jwtString must be provided in jsonWebToken.verify"));
+  if (!PEMKey || PEMKey === "")
+    return done(new Error("PEMKey must be provided in jsonWebToken.verify"));
 
   // asynchronous wrapper for callback
-  var done = function() {
-    var args = Array.prototype.slice.call(arguments, 0);
+  const done = function() {
+    const args = Array.prototype.slice.call(arguments, 0);
     return process.nextTick(function() {
       callback.apply(null, args);
     });
   };
 
   // make sure we have the required fields in options
-  if (!(options.audience && (typeof options.audience === 'string' || 
+  if (!(options.audience && (typeof options.audience === "string" || 
     (Array.isArray(options.audience) && options.audience.length > 0))))
-    return done(new Error('invalid options.audience value is provided in jsonWebToken.verify'));
+    return done(new Error("invalid options.audience value is provided in jsonWebToken.verify"));
   if (!options.algorithms)
-    return done(new Error('options.algorithms is missing in jsonWebToken.verify'));
-  if (!Array.isArray(options.algorithms) || options.algorithms.length == 0 ||
-    (options.algorithms.length === 1 && options.algorithms[0] === 'none'))
-    return done(new Error('options.algorithms must be an array containing at least one algorithm'));
+    return done(new Error("options.algorithms is missing in jsonWebToken.verify"));
+  if (!Array.isArray(options.algorithms) || options.algorithms.length === 0 ||
+    (options.algorithms.length === 1 && options.algorithms[0] === "none"))
+    return done(new Error("options.algorithms must be an array containing at least one algorithm"));
 
-  /*********************************************************************
+  /**
```

Comment:
Formatting for these breaks is incorrect.  There are several more of these below in this file.